aml/org.testeditor.aml.model/src/org/testeditor/aml/ModelUtil.xtend

```diff
@@ -107,4 +108,27 @@ class ModelUtil {
 		}
 	}
 
+	/**
+	 * @return the type returned by the fixture of the given interaction (may be null)
+	 */
+	def JvmTypeReference getReturnType(InteractionType interaction) {
+		interaction.defaultMethod?.operation?.returnType
+	}
+
+	/**
+	 * @return whether the given jvmTypeReference is assignable to clazz (without widening primitive types)
+	 */
+	def boolean isAssignableWithoutWidening(Class<?> clazz, JvmTypeReference jvmTypeReference) {
```

Comment:
What I meant is that primitive types (like int) will be assignable only if assigned to the exact same type (thus assigning an int to long won't be possible)...